Lib/sqlite3/__main__.py

```diff
@@ -0,0 +1,92 @@
+import sqlite3
+import sys
+
+from argparse import ArgumentParser
+from code import InteractiveConsole
+from textwrap import dedent
+
+
+class SqliteInteractiveConsole(InteractiveConsole):
+
+    def __init__(self, database):
+        super().__init__()
+        self._con = sqlite3.connect(database, isolation_level=None)
```

Comment:
Hmm, out of curiosity, don't we have to care about the connection lifecycle?

Reply:
`dealloc` will implicitly close the connection, but we might as well do it explicitly, since that's good practice.

Reply:
Added an explicit close.